exercises/meetup/test/Main.purs

```diff
@@ -0,0 +1,395 @@
+module Test.Main where
+
+import Prelude
+import Test.Unit.Assert as Assert
+import Control.Monad.Eff (Eff)
+import Data.Date (Weekday(..), canonicalDate)
+import Data.Enum (toEnum)
+import Data.Maybe (Maybe(..))
+import Meetup (Week(..), meetup)
+import Test.Unit (suite, test)
+import Test.Unit.Main (runTest)
+
+main :: Eff _ Unit
+main = runTest do
+  suite "Meetup.meetup" do
+
+    test "monteenth of May 2013" $
+      Assert.equal (Just $ canonicalDate <$> toEnum 2013 <*> toEnum 5 <*> toEnum 13) 
+                   (meetup <$> toEnum 2013 <*> toEnum 5 <@> Teenth <@> Monday)
```

Comment:
@lpil They are all aliases in Prelude:

```
infixl 4 map as <$>
infixl 4 apply as <*>
infixl 4 flap as <@>
```

Using [pursuit](https://pursuit.purescript.org/) instead of google makes it easier. Search with parentheses for operators. Here are the links:

* `<$>`: [pursuit](https://pursuit.purescript.org/packages/purescript-prelude/2.4.0/docs/Data.Functor#v:(<$>))
* `<*>`: [pursuit](https://pursuit.purescript.org/packages/purescript-prelude/2.4.0/docs/Control.Apply#v:(<*>))
* `<@>`: [pursuit](https://pursuit.purescript.org/packages/purescript-prelude/2.4.0/docs/Data.Functor#v:(<@>))